app/assets/javascripts/student_profile_v2/chart.js

```diff
@@ -0,0 +1,77 @@
+(function() {
+  window.shared || (window.shared = {});
+  var dom = window.shared.ReactHelpers.dom;
+  var createEl = window.shared.ReactHelpers.createEl;
+  var merge = window.shared.ReactHelpers.merge;
+
+  var ProfileChartSettings = window.ProfileChartSettings;
+  var HighchartsWrapper = window.shared.HighchartsWrapper;
+
+  // Component for all charts in the profile page.
+  window.shared.Chart = React.createClass({
+    displayName: 'Chart',
+
+    propTypes: {
+      series: React.PropTypes.arrayOf( // you can plot multiple series on the same graph
+        React.PropTypes.shape({
+          name: React.PropTypes.string.isRequired, // e.g. 'Scaled score'
+          data: React.PropTypes.array.isRequired // [year, month, date, value] quads
+        })
+      ),
+      title: React.PropTypes.string.isRequired, // e.g. 'MCAS scores, last 4 years'
```

Comment:
`title` is a Highcharts options (where you can pass the text and alignment or other options), but this component only allows modifying the text.  So I'd suggest making that explicit with `titleText` or something like that.